portal/src/graphql/portal/PasswordlessAuthenticatorScreen.tsx

```diff
@@ -121,7 +121,12 @@ const PasswordlessAuthenticatorScreen: React.FC = function PasswordlessAuthentic
     );
 
     // TODO: handle error
-    updateAppConfig(newAppConfig).catch(() => {});
+    updateAppConfig(newAppConfig)
```

Comment:
Templates will be updated by separate mutation? I think update app config and update template can be performed in parallel.